config/src/config/api_config.rs

```diff
@@ -42,6 +42,10 @@ pub struct ApiConfig {
 
     /// Max gas unit for view function.
     pub max_gas_view_function: u64,
+
+    /// Performance functionality
+    pub max_runtime_workers: Option<usize>,
+    pub runtime_worker_amplifier: usize,
```

Comment:
I would call this runtime_worker_multiplier.

Amplifier makes it sound like you always want to increase this value.

Reply:
Done, changed to: `runtime_worker_multiplier`.